app/templates/createclasssuccess.html

```diff
@@ -0,0 +1,28 @@
+﻿{% extends "base.html" %}
+{% block style %}
+<link rel="stylesheet" href="../static/content/createstyle.css">
+{% endblock %}
+{% block content %}
+<div class="create-class-wrapper">
+    <div class="create-container">
+
+        <form action="" method="post" novalidate class="create-form">
```

Comment:
Why is there a POST request to the site itself? This will be called as `createclasssuccess`, which shouldn't have any POST request to it.

Reply:
noted